config/routes.rb

```diff
@@ -14,6 +14,9 @@
           number: /#{Gem::Version::VERSION_PATTERN}(?=\.json\z)|#{Gem::Version::VERSION_PATTERN}/
         }
       end
+      get '/versions' => 'compact_index#versions'
+      get '/info/:gem_name' => 'compact_index#info', as: :info
+      get '/names' => 'compact_index#names'
```

Comment:
These need to be mounted at the root not inside `/api/v2` to match what we use right now.